docs/core-concepts/working-with-templates/responsive-images.md

```diff
@@ -143,6 +154,9 @@ Here's an example of how you can combine that technique with a focal point:
 {% endif %}
 ```
 
+You should decide if you want to show you image as `background-image` or in an img tag with `src` and `srcset`.
```

Comment:
This needs more thinking about and a solid recommendation that is reasonable for accessibility.

Reply:
I'm not sure I fully understand what the message is here. If you aren't using `background-image` and `background-position` you would need to be using `object-fit` and `object-position` to use the focal point information. If that is what you are getting at it should be more clear.